src/shared/sources/au/index.js

```diff
@@ -62,7 +67,63 @@ module.exports = {
 
         const indexForCases = dataKeysByColumnIndex.findIndex(key => key === 'cases')
         const casesFromTotalRow = parse.number(
-          normalizedTable.find(row => row.some(column => column === 'Total'))[indexForCases]
+          normalizedTable.find((row) => row.some(column => column === 'Total'))[indexForCases]
+        )
+        assertTotalsAreReasonable({ computed: summedData.cases, scraped: casesFromTotalRow })
+        return states
+      }
+    },
+    {
+      startDate: '2020-04-02',
+      crawl: [
+        {
+          type: 'headless',
+          data: 'table',
+          url: 'https://www.health.gov.au/resources/total-covid-19-cases-and-deaths-by-states-and-territories'
+        }
+      ],
+      scrape ($, date, { assertTotalsAreReasonable, getSchemaKeyFromHeading, normalizeTable }) {
+        const normalizedTable = normalizeTable({ $, tableSelector: '.ng-scope table' })
```

Comment:
As I mentioned in Slack, I'm not sure what these helper methods do (eg 'schemaKeysByHeadingFragment').  I have a feeling it's good :-) though it does make the code more opaque.

Reply:
Lets workshop some renames for these utils once you have a read of the JSdoc and get a clearer picture of what they're for? Definitely want to make the naming clear for folks :)